app/views/reports/index.html.erb

```diff
@@ -15,13 +15,13 @@
   </div>
 
   <div class="form-group">
-    <%= f.label :completed %>
-    <%= f.select(:complete_eq, [['Yes', true], ['No', false]], {}, { :class => 'form-control' }) %>
+    <%= f.label "Limit to complete records (returned to origin)" %>
+    <%= f.check_box :complete_eq, {}, true, false %>
```

Comment:
@rishitejar : Can we put the check_box first, before the label.  That way things line up better.